File: php/testcode/benchmark_test_00509.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00509(BenchmarkRequest $req): BenchmarkResponse {
    $class = $req->param('class');
    $method = $req->param('method');
    $class::$method();
    return BenchmarkResponse::ok('dispatched');
}

File: rust/testcode/benchmark_test_00509.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let url = req.param("url");
    let client = reqwest_danger_client(true);
    let result = format!("Fetched {} with client {}", url, client);
    super::shared::BenchmarkResponse::ok(&result)
}
fn reqwest_danger_client(accept_invalid: bool) -> String {
    format!("client(danger_accept_invalid_certs={})", accept_invalid)
}
